lib/ast-walk.js

```diff
@@ -0,0 +1,52 @@
+var parse = require('acorn').parse;
+
+module.exports = function (src, opts) {
+    if (!opts) opts = {}
+    var ast = src;
+    if (typeof src === 'string') {
+        try {
+            ast = parse(src, {
+                ecmaVersion: 9,
+                allowReturnOutsideFunction: true
+            })
+        }
+        catch (err) { ast = parse('(' + src + ')') }
```

Comment:
@Ramshackle-Jamathon - can you add in a comment about this catch? Why its needed and what value it provides?

Reply:
@mcwhittemore I ahh shoulda mentioned that this was just a copypasta of the astw dependency. But it makes total sense to tailor it to our use case since we can!